webpack.config.js

```diff
@@ -151,7 +151,7 @@ const webpackConfig = {
 	},
 	resolve: {
 		extensions: [ '.json', '.js', '.jsx' ],
-		modules: [ path.join( __dirname, 'client' ), 'node_modules' ],
```

Comment:
👏

Reply:
Actually I think we can remove this option altogether now since its the default in webpack. Not a biggy though.